package.json

```diff
@@ -58,6 +58,9 @@
     "lerna-changelog": "2.2.0",
     "typescript": "4.3.5"
   },
+  "overrides":{
+    "@types/bson": "4.0.5"
+  },
```

Comment:
This needs to also be present in the `package.json` for the mongo instrumentation, so if lerna hoisting is not applied the correct version is still used